Mathlib/Data/Rel.lean

```diff
@@ -114,6 +115,24 @@ theorem comp_left_id (r : Rel α β) : @Eq α • r = r := by
   simp
 #align rel.comp_left_id Rel.comp_left_id
 
+@[simp]
+theorem comp_right_bot (r : Rel α β) : r • (⊥ : Rel β γ) = ⊥ := by
+  ext x y
+  simp[comp, Bot.bot]
+
+@[simp]
+theorem comp_left_bot (r : Rel α β) : (⊥ : Rel γ α) • r = ⊥ := by
+  ext x y
+  simp[comp, Bot.bot]
+
+theorem comp_right_top (r : Rel α β) : r • (⊤ : Rel β γ) = λ x _ ↦ x ∈ r.dom := by
```

Comment:
It's not so clear to me whether this is indeed a useful `@[simp]` lemma, but I'm going to lean in the direction of more simplification.
```suggestion
@[simp]
theorem comp_right_top (r : Rel α β) : r • (⊤ : Rel β γ) = λ x _ ↦ x ∈ r.dom := by
```